frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoService.java

```diff
@@ -31,6 +37,20 @@ public class GoService extends Service {
 
     private final int notificationId = 8;
 
+    private ViewModelStoreOwner viewModelStoreOwner;
+
+    private final BroadcastReceiver usbStateReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            String action = intent.getAction();
+            if (viewModelStoreOwner != null && UsbManager.ACTION_USB_DEVICE_DETACHED.equals(action)) {
```

Comment:
`viewModelStoreOwner` never seems to be initialized, is there some piece missing?

Reply:
Wow, true and very confusing, as I tested it and @thisconnect did too :thinking: I'll check again later to see what went wrong.